### ISSUE TITLE
Error with install with v3.2

### ISSUE THREAD

**Issue by sharath20284** [kbenoit/quanteda#88]
Hello,
I am newbie, so please point me to solution if this is resolved.
I am getting an error as below when trying to install quanteda in RStudio on Windows.

library(devtools)
> if (!require(quanteda)) install_github("kbenoit/quanteda")
Loading required package: quanteda
Downloading GitHub repo kbenoit/quanteda@master
Installing quanteda
"C:/PROGRA~1/R/R-32~1.3/bin/x64/R" --no-site-file --no-environ --no-save --no-restore CMD INSTALL  \
  "C:/Users/E480731/AppData/Local/Temp/RtmpOgNoTV/devtools73782e8e4229/kbenoit-quanteda-6eb87dd"  \
  --library="C:/Users/E480731/Documents/R/win-library/3.2" --install-tests 

* installing *source* package 'quanteda' ...
** libs

*** arch - i386
Warning: running command 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-32~1.3/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-32~1.3/share/make/winshlib.mk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="quanteda.dll" OBJECTS="RcppExports.o ngrams.o wordfish.o"' had status 127
ERROR: compilation failed for package 'quanteda'
* removing 'C:/Users/Sharath/Documents/R/win-library/3.2/quanteda'

**Comment by kbenoit** [comment]
No worries!  Did you install RTools as per the instructions for Windows?

On 8 Jan 2016, at 18:29, sharath20284 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello,
I am newbie, so please point me to solution if this is resolved
I am getting an error as below when trying to install quanteda in RStudio on Windows

library(devtools)

if (!require(quanteda)) install_github("kbenoit/quanteda")
Loading required package: quanteda
Downloading GitHub repo kbenoit/quanteda@master
Installing quanteda
"C:/PROGRA~1/R/R-32~13/bin/x64/R" --no-site-file --no-environ --no-save --no-restore CMD INSTALL \
"C:/Users/E480731/AppData/Local/Temp/RtmpOgNoTV/devtools73782e8e4229/kbenoit-quanteda-6eb87dd" \
--library="C:/Users/E480731/Documents/R/win-library/32" --install-tests

  *   installing source package 'quanteda' ** libs

*** arch - i386
Warning: running command 'make -f "Makevarswin" -f "C:/PROGRA~1/R/R-32~13/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-32~13/share/make/winshlibmk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="quantedadll" OBJECTS="RcppExportso ngramso wordfisho"' had status 127
ERROR: compilation failed for package 'quanteda'

  *   removing 'C:/Users/Sharath/Documents/R/win-library/32/quanteda'

**Comment by sharath20284** [comment]
Thanks!

Yes,  I had tried to install Rtools earlier, but failed
Warning in install.packages :
  package ‘Rtools’ is not available (for R version 3.2.3)

I had to download Rtools.exe  from https://cran.r-project.org/bin/windows/Rtools/ and install it.

Quanteda install completed successfully.

**Comment by kbenoit** [comment]
That’s it!  Glad it’s working.


On 8 Jan 2016, at 18:56, sharath20284 <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks!

Yes, I had tried to install Rtools earlier, but failed
Warning in install.packages :
package ‘Rtools’ is not available (for R version 3.2.3)

I had to download Rtools.exe from https://cran.r-project.org/bin/windows/Rtools/ and install it.

Quanteda install completed successfully.
